examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p>Choose more specific title, e.g. "Learning rate schedulers comparison".</p><p> </p><p>Also you are mixing "decay" and "learning rate scheduler" namings throughout the notebook. Choose one for consistency. Do not forget about the notebook name.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
There's still an inconsistency between the name of the notebook and its title. You should either defend your point or stick to suggested changes, but **under no circumstances** mark things you ignored as _resolved_.